<launch>
  <arg name="debug" default="false"/>

<!--   <param name="warehouse_host" value="localhost"/>
  <param name="warehouse_port" value="27017"/>  27017 is default, use a private one to avoid error 48 conflicts
  <node  pkg="warehouse_ros" type="mongo_wrapper_ros.py" name="mongo">
    <param name="database_path" value="warehouse_data"/>
    <param name="overwrite" value="false"/>
  </node> -->

  <node pkg="world_canvas_server" type="world_canvas_server" name="world_canvas_server" args="$(arg debug)" output="screen">
    <param name="start_map_manager" value="true"/>
    <param name="auto_save_map" value="false"/>
  </node> 
</launch>
